src/test/java/seedu/address/model/food/MacronutrientTest.java

```diff
@@ -0,0 +1,55 @@
+package seedu.address.model.food;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+class MacronutrientTest {
+
+    private static class MacronutrientObjectCreator extends Macronutrient {
+        MacronutrientObjectCreator(String name, int amount, int caloriMultiplier) {
```

Comment:
typo in `caloriMultiplier`.

Also I think you `MacronutrientStub` or something would be a better class name. The current one sounds like this is an object that creates other macronutrients.

Reply:
Ok I will change it that way